src/ui/components/DashKit/plugins/Control/Control.tsx

```diff
@@ -583,7 +549,14 @@ class Control extends React.PureComponent<PluginControlProps, PluginControlState
                 }
                 break;
             case LOAD_STATUS.FAIL: {
-                return this.renderError();
+                return (
+                    <Error
+                        errorData={this.state.errorData}
+                        onClickRetry={() => {
```

Comment:
Move anonymous function as method, or this component will re-render every time it's created. 
```
onClickRetry = () => this.reload();
```
And then
```
onClickRetry={this.onClickRetry}
```